docs/refman/loaderyaml.py

```diff
@@ -174,7 +273,13 @@ def as_methods(mlist: T.List[Function]) -> T.List[Method]:
 
         methods = raw.pop('methods', [])
         obj = Object(methods=[], obj_type=obj_type, **raw)
-        obj.methods = as_methods([self._process_function_base(x, obj) for x in methods])
+
+        methods = []
+        for x in methods:
```

Comment:
Ummm.

This deleted `methods`, replaced it with an empty list, and then iterated over the empty list, mutated values, and appended to the thing it was iterating over.

Reply:
Fixed via 8d648532295d131898b392032d13cd1b13e639af.

Reply:
:cry: